tests/python/test_cli.py

```diff
@@ -172,8 +172,7 @@ def test_cli_video():
 
 
 def test_cli_format():
-    with patch_sys_argv_helper(["ti", "format", "-d",
-                                "51fff7af"]) as custom_argv:
+    with patch_sys_argv_helper(["ti", "format", "51fff7af"]) as custom_argv:
```

Comment:
@rexwangcc Sorry about breaking your test in #1127!

Reply:
Sorry I totally forgot there's a test for it when looking at the PR :P